libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -351,6 +351,8 @@ singleton AP_Vehicle method reboot void boolean
 singleton AP_Vehicle method is_landing boolean
 singleton AP_Vehicle method is_taking_off boolean
 singleton AP_Vehicle method set_crosstrack_start boolean Location
+singleton AP_Vehicle method set_target_rate_and_throttle boolean float'skip_check float'skip_check float'skip_check float'skip_check
```

Comment:
I'd move this higher in the .desc to be grouped with the others like set_target_angle_and_climbrate

Reply:
Done